client/components/date-range/index.js

```diff
@@ -477,7 +479,11 @@ export class DateRange extends Component {
 					<div className="date-range__popover-inner">
 						<div className="date-range__controls">
 							{ this.props.renderHeader( headerProps ) }
-							{ this.renderDateHelp() }
+							{ this.props.customTitle ? (
```

Comment:
Could we add the customTitle support but not to remove renderDateHelp? I find the link quite useful. Also if we are going to hide that, it feels like a separate question 🤔

Reply:
That's a good point, and maybe worth a discussion because in the designs there is no renderDateHelp. I agree it's useful though, but this PR was just aligning up with the designs in Figma as to what's shown in this area. Maybe someone from design can weigh in here before we take this any further? cc: @ederrengifo

Reply:
>I find the link quite useful

Do you mean the reset selection link?